contracts/src/oracle_implementation/OracleImplementation.cairo

```diff
@@ -3,8 +3,8 @@
 from starkware.cairo.common.cairo_builtins import HashBuiltin
 from starkware.cairo.common.bool import TRUE, FALSE
 
-from contracts.entry.structs import Entry
-from contracts.oracle_implementation.library import (
+from entry.structs import Entry
+from oracle_implementation.library import (
```

Comment:
This is great. In the future we can move this into src/empiric/... or similar for beautiful imports, also for others importing our library